packages/underlay/src/Underlay.ts

```diff
@@ -22,16 +22,40 @@ import styles from './underlay.css.js';
 
 /**
  * @element sp-underlay
+ *
+ * @fires close - When the underlay is "clicked" and the consuming pattern should chose whether to close based on that interaction
  */
 export class Underlay extends SpectrumElement {
     public static override get styles(): CSSResultArray {
         return [styles];
     }
 
+    private canClick = false;
+
     @property({ type: Boolean, reflect: true })
     public open = false;
 
+    public override click(): void {
+        this.dispatchEvent(new Event('close'));
+    }
+
+    protected handlePointerdown(): void {
+        this.canClick = true;
+    }
+
+    protected handlePointerup(): void {
+        if (this.canClick) {
+            this.dispatchEvent(new Event('close'));
```

Comment:
Another option here could be `this.click()` - today the lines are exactly the same, but sending both paths through the same method would guarantee that the behavior of a user-driven "click" and a code-driven "click" stay the same.

Reply:
Yes!